README.md

```diff
@@ -31,6 +31,45 @@ Test data is freely available from [OpenSlide](http://openslide.cs.cmu.edu/downl
 
 To see all available options, run: `wsi2dcm --help`.
 
+## Complite set of parametes
+
+##### input
```

Comment:
Can we improve these explanations in a similar fashion to the code comments we made in the last pull request? Also, let's use full sentences with proper punctuation.

Reply:
done, i think